src/ui/molecules/stepper/Stepper.tsx

```diff
@@ -203,7 +203,9 @@ export const Stepper: React.FC<StepperProps> = ({
                   <Typography
                     fontSize="x-small"
                     fontWeight={
-                      step.id === currentStepId && step.status !== 'completed' ? 'bold' : 'light'
+                      step.id === currentStepId && getStepStatus(step) !== 'completed'
```

Comment:
I don't know if it's important but I would have taken this piece of code out of the JSX to improve the readability